package.json

```diff
@@ -29,15 +29,14 @@
   },
   "homepage": "https://github.com/Microsoft/typed-rest-client#readme",
   "devDependencies": {
-    "@types/mocha": "^2.2.44",
-    "@types/node": "^6.0.92",
+    "@types/mocha": "^9.1.1",
+    "@types/node": "^16.11.42",
     "@types/shelljs": "0.7.4",
-    "mocha": "^3.5.3",
+    "mocha": "^9.2.2",
```

Comment:
Should Mocha support working through node 10 in pipelines? If so, it's better to change the version to 6.2.3 to prevent issues with compatibility (see breaking changes) - https://github.com/mochajs/mocha/releases/tag/v9.0.0